### PR TITLE
Added Portuguese translation to the repository exceptions

### PR DESCRIPTION
**Pull request by pedroresende**
This PR extends the EZP-24793 - Make Repository exceptions translatable story to include the Portuguese translation

### CODE REVIEW

**Comment by yannickroger** [comment]
@pedroresende you just need to ping some Portuguese speakers for a review :)

**Comment by lolautruche** [comment]
+1, assuming the translation is correct :smile:

File: eZ/Bundle/EzPublishCoreBundle/Resources/translations/repository_exceptions.pt.xlf [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+            </trans-unit>
+            <!-- NotFoundException -->
+            <trans-unit id="ad3acb293dfd5d85893a49e9b0d69a3b">
+                <source>Could not find '%what%' with identifier '%identifier%'</source>
+                <target>Não é possível encontrar '%what%' com o identificador '%identifier%'</target>
+            </trans-unit>
+            <!-- FieldTypeNotFoundException -->
+            <trans-unit id="71e12d2a6eae4e348b100502950a54bc">
+                <source>FieldType '%fieldType%' not found, needs to be implemented or configured to use FieldType\\Null\\Type (%ezpublish.fieldType.eznull.class%)</source>
+                <target>O FieldType '%fieldType%' não foi encontrado, é necessário implementa-lo ou configura-lo para usar FieldType\\Null\\Type (%ezpublish.fieldType.eznull.class%)</target>
```
Comment (pcardiga) [inline eZ/Bundle/EzPublishCoreBundle/Resources/translations/repository_exceptions.pt.xlf]: Nitpick: "implementá-lo" and "configurá-lo", with an "á"

File: eZ/Bundle/EzPublishCoreBundle/Resources/translations/repository_exceptions.pt.xlf [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+            </trans-unit>
+            <!-- LimitationNotFoundException -->
+            <trans-unit id="8a75362817e021200bf2d5df8b3d5374">
+                <source>Limitation '%limitation%' not found, needs to be implemented or configured to use Limitation\\BlockingLimitationType (%ezpublish.api.role.limitation_type.blocking.class%)</source>
+                <target>A limitação '%limitation%' não foi encontrada, not found, é necessário implementa-la ou configura-la para usar Limitation\\BlockingLimitationType (%ezpublish.api.role.limitation_type.blocking.class%)</target>
```
Comment (pcardiga) [inline eZ/Bundle/EzPublishCoreBundle/Resources/translations/repository_exceptions.pt.xlf]: Same as above
Nitpick: "implementá-la" and "configurá-la"

File: eZ/Bundle/EzPublishCoreBundle/Resources/translations/repository_exceptions.pt.xlf [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+                <target>O argumento '%argumentName%' é inválido: '%actualValue%' está errado</target>
+            </trans-unit>
+            <trans-unit id="f956ff7b4e7ca4cca6ae8d1fb74b9c70">
+                <source>Argument '%argumentName%' is invalid: '%actualValue%' is wrong value in class '%className%'</source>
+                <target>O argumento  '%argumentName%' é inválido: '%actualValue%' está errado na classe '%className%'</target>
+            </trans-unit>
+            <!-- LimitationValidationException -->
+            <trans-unit id="f3f16fa96131bd451f91a6c5fa26eae8">
+                <source>Limitations did not validate</source>
+                <target>As limitações não validam</target>
```
Comment (pcardiga) [inline eZ/Bundle/EzPublishCoreBundle/Resources/translations/repository_exceptions.pt.xlf]: Might be better "As limitações não são válidas"

**Comment by pedroresende** [comment]
@pcardiga thanks for the review, fixed and rebased

**Comment by miguelcleverti** [comment]
+1

**Comment by pcardiga** [comment]
Other than nitpicks, +1

File: eZ/Bundle/EzPublishCoreBundle/Resources/translations/repository_exceptions.pt.xlf [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+                <target>O FieldType '%fieldType%' não foi encontrado, é necessário implementá-lo ou configura-lo para usar FieldType\\Null\\Type (%ezpublish.fieldType.eznull.class%)</target>
+            </trans-unit>
+            <!-- LimitationNotFoundException -->
+            <trans-unit id="8a75362817e021200bf2d5df8b3d5374">
+                <source>Limitation '%limitation%' not found, needs to be implemented or configured to use Limitation\\BlockingLimitationType (%ezpublish.api.role.limitation_type.blocking.class%)</source>
+                <target>A limitação '%limitation%' não foi encontrada, not found, é necessário implementá-la ou configurá-la para usar Limitation\\BlockingLimitationType (%ezpublish.api.role.limitation_type.blocking.class%)</target>
+            </trans-unit>
+            <!-- UnauthorizedException -->
+            <trans-unit id="cb68b93f274ad087f7188c8745ef5fee">
+                <source>User does not have access to '%function%' '%module'</source>
```
Comment (lolautruche) [inline eZ/Bundle/EzPublishCoreBundle/Resources/translations/repository_exceptions.pt.xlf]: I need to fix `%module` (missing a `%`)

**Comment by lolautruche** [comment]
Can you please rebase and fix the translations for UnauthorizedExceptions please?

**Comment by pedroresende** [comment]
@lolautruche sure, don't forget the module on line 108/109

**Comment by pedroresende** [comment]
updated and rebased

**Comment by andrerom** [comment]
Cool, any notes on how you did it so we can get others started as well? :)

**Pull request merged**